dali/imgcodec/decoders/nvjpeg2k/nvjpeg2k.cc

```diff
@@ -36,11 +37,12 @@ bool check_status(nvjpeg2kStatus_t status, ImageSource *in) {
 }
 }  // namespace
 
-NvJpeg2000DecoderInstance::NvJpeg2000DecoderInstance(int device_id)
-: BatchParallelDecoderImpl(device_id)
+NvJpeg2000DecoderInstance::NvJpeg2000DecoderInstance(
+    int device_id, const std::map<std::string, any> &params)
+: BatchParallelDecoderImpl(device_id, params)
```

Comment:
I know this is out of scope, but this alignment of constructor initializer list is both ugly and against GSG as far as I rememeber.

Reply:
I can gladly abandon it the same day we put opening braces (at least for functions) on a new line ;)